sdk-src/src/main/java/com/hpe/adm/nga/sdk/Query.java

```diff
@@ -16,43 +16,43 @@
- *   {@link #not(String, QueryMethod, Object)} methods.
- *   </p>
- *   <p>
- *   See the REST API documentation for information as to how to build queries.  This API supports building queries in a simplified Builder pattern
- *   </p>
- *   <p>
- *       Each query statement consists of the field name, query method and the object of the query.  The object can be a simple primitive or a further {@code Query} object
- *   </p>
- *   <p>
- *       For example:
- *       <br>
- *       To build a query such as {@code name eq 'entity_name'} the following code is used:
- *       <br><code>Query.statement("name", QueryMethod.EqualTo, "entity_name");</code><br>
- *       <br>
- *       To negate this statement such as {@code !name eq 'entity_name'} the following code is used:
- *       <br><code>Query.not("name", QueryMethod.EqualTo, "entity_name");</code><br>
- *   </p>
- *   <p>
- *       These statements produce a {@code QueryBuilder} object.  By calling the {@link QueryBuilder#build()} method this creates a {@code Query object} which can then
- *       be used in conjunction with entity or other contexts
- *   </p>
- *   <p>
- *       {@code Query} objects can be chained together (and/or) by using the correct methods such as {@link QueryBuilder#and(QueryBuilder)} or
- *       {@link QueryBuilder#orNot(String, QueryMethod, Object)}
- *       <br>
- *       {@link QueryBuilder#build()} should be called once the entire query has been built
- *   </p>
+ * The class to build complex queries.  An instance of this class is created using the {@link #statement(String, QueryMethod, Object)} or
+ * {@link #not(String, QueryMethod, Object)} methods.
+ * </p>
+ * <p>
+ * See the REST API documentation for information as to how to build queries.  This API supports building queries in a simplified Builder pattern
+ * </p>
+ * <p>
+ * Each query statement consists of the field name, query method and the object of the query.  The object can be a simple primitive or a further {@code Query} object
+ * </p>
+ * <p>
+ * For example:
+ * <br>
+ * To build a query such as {@code name eq 'entity_name'} the following code is used:
+ * <br><code>Query.statement("name", QueryMethod.EqualTo, "entity_name");</code><br>
+ * <br>
+ * To negate this statement such as {@code !name eq 'entity_name'} the following code is used:
+ * <br><code>Query.not("name", QueryMethod.EqualTo, "entity_name");</code><br>
+ * </p>
+ * <p>
+ * These statements produce a {@code QueryBuilder} object.  By calling the {@link QueryBuilder#build()} method this creates a {@code Query object} which can then
+ * be used in conjunction with entity or other contexts
+ * </p>
+ * <p>
+ * {@code Query} objects can be chained together (and/or) by using the correct methods such as {@link QueryBuilder#and(QueryBuilder)} or
+ * {@link QueryBuilder#orNot(String, QueryMethod, Object)}
+ * <br>
+ * {@link QueryBuilder#build()} should be called once the entire query has been built
+ * </p>
  */
```

Comment:
We should agree on formatting principles otherwise these changes will come up every time